sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -45,7 +45,7 @@ public boolean isEnabled() {
     return enabled;
   }
 
-  SentryClient(final @NotNull SentryOptions options) {
+  public SentryClient(final @NotNull SentryOptions options) {
```

Comment:
Should we mark it internal so people think twice before using it unless instructed to do so?

Reply:
Yeah, makes sense!